### PR TITLE
Update table UI into cards layout

### PR DESCRIPTION
**Pull request by lonamiaec**
This is going to be used for a test on the Lienzo pages to see if this layout performs better than the responsible table UI

The `<table>`  doesn't scale well on mobile views and requires users to scroll horizontally to see all of the data. This is a design aimed at solving that issue.

image

### CODE REVIEW

**Comment by keyvanakbary** [comment]
Really, really cool 👌

File: src/comparison-table/ComparisonTable.less [hunk 1]
```diff
@@ -29,4 +33,26 @@
...
+  }
+
+  .card-item {
+    border-bottom: 1px solid #e2e6e8 !important;
+    border-left: 1px solid #e2e6e8 !important;
+    border-right: 1px solid #e2e6e8 !important;
+  }
+
+  .last-card-item {
+    border-radius: 4px;
```
Comment (cassiogreco) [inline src/comparison-table/ComparisonTable.less]: Just saw this in the images. The bottom-most item in the card has top border-radius (breaking the design). We should have the border-radius only in the bottom part, so we should use `border-bottom` instead

File: src/comparison-table/ComparisonTable.js [hunk 2]
```diff
@@ -100,25 +101,35 @@ export default class ComparisonTable extends Component {
...
+          <div className="table-responsive hidden-xs">
+            <table className="table table-bordered">
+              <thead>
+                <tr className="bg-primary text-inverse">
+                  <th>Provider</th>
+                  <th className="hidden-md hidden-lg hidden-xl text-xs-right text-xs-nowrap">Amount received</th>
+                  <th className="hidden-md hidden-lg hidden-xl text-xs-right text-xs-nowrap">Exchange rate</th>
+                  <th className="text-xs-right text-xs-nowrap">Service fee</th>
+                  <th className="hidden-xs hidden-sm text-xs-right hidden-xs text-xs-nowrap">Exchange rate</th>
+                  <th className="hidden-xs hidden-sm text-xs-right text-xs-nowrap">Amount received</th>
```
Comment (cassiogreco) [inline src/comparison-table/ComparisonTable.js]: Didn't think of this before. We can remove some of these styles, as we are applying `hidden-xs` to the table itself, no need to have hidden-xs on the headers.

Just keep in mind that in small screens we are shifting the order of the columns, so I believe only the `hidden-xs` need to be removed

File: src/comparison-table/ComparisonTable.js [hunk 3]
```diff
@@ -100,25 +101,35 @@ export default class ComparisonTable extends Component {
...
+              </thead>
+              <tbody>
+                {providers.map(p => {
+                  return <Provider key={p.id} name={p.name} logo={p.logo} fees={p.fees} rate={p.rate} hiddenFees={p.hiddenFees} maxFee={p.maxFee} source={this.props.source} target={this.props.target} amount={p.amount} highestAmount={providers[0].amount} collectedAt={p.collectedAt}/>
+                })}
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <div className="hidden-sm hidden-xl hidden-lg hidden-md m-b-2">
```
Comment (cassiogreco) [inline src/comparison-table/ComparisonTable.js]: nit - just to make it easier to read, can we keep the `hidden` in order? i.e. `sm`, then `md`, then `lg`, then `xl`

File: src/comparison-table/ComparisonTable.less [hunk 4]
```diff
@@ -29,4 +33,26 @@
...
+
+    a {
+      text-decoration: none;
+    }
+  }
+
+  .card-item {
+    border-bottom: 1px solid #e2e6e8 !important;
+    border-left: 1px solid #e2e6e8 !important;
+    border-right: 1px solid #e2e6e8 !important;
```
Comment (cassiogreco) [inline src/comparison-table/ComparisonTable.less]: didn't mention this before as well - but we can substitute the hex value with a variable - you can look in the `variables` file inside of bootstrap and search for the color code and then just use `@<color-name>`.

We can change it in the `.card-header` as well

**Comment by cassiogreco** [comment]
Awesome! 👍 

Let's test it!

**Comment by kdelemme** [comment]
Like the design 👍  super neat!

**Pull request merged**